### ISSUE TITLE
Error when building Docker image

### ISSUE THREAD

**Issue by jpslopes** [facebook/duckling#559]
Hello,

Duckling is raising the following exception when building the Docker image:
```console
Preprocessing executable 'duckling-example-exe' for duckling-0.1.6.1..
Building executable 'duckling-example-exe' for duckling-0.1.6.1..
[1 of 2] Compiling Duckling.Data.TimeZone
[2 of 2] Compiling Main

/duckling/exe/ExampleMain.hs:108:27: error:
    • Variable not in scope:
        parseDimensions :: Text -> Maybe (Seal Dimension)
    • Perhaps you meant one of these:
        ‘parseDimension’ (line 120),
        ‘allDimensions’ (imported from Duckling.Dimensions)
    |
108 |           pure $ mapMaybe parseDimensions txtDims
    |                           ^^^^^^^^^^^^^^^
Completed 63 action(s).

--  While building package duckling-0.1.6.1 (scroll up to its section to see the error) using:
      /root/.stack/setup-exe-cache/x86_64-linux-tinfo6/Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.5 --builddir=.stack-work/dist/x86_64-linux-tinfo6/Cabal-2.4.0.1 build lib:duckling exe:custom-dimension-example exe:duckling-example-exe exe:duckling-expensive exe:duckling-regen-exe exe:duckling-request-sample --ghc-options ""
    Process exited with code: ExitFailure 1
```

It looks like that it is the same exception as here: https://github.com/facebook/duckling/runs/1553658600

Thanks

**Comment by hrabago** [comment]
I had the same issue building with stack...

**Comment by chessai** [comment]
Looks like our internal CI doesn't actually build the example application. I opened an issue to track that (#561) and a PR to fix the build error you're experiencing (#560)
